pkg/secrets/no_secrets.go

```diff
@@ -17,7 +17,7 @@ import (
 var SecretBackendOutputMaxSize = 1024 * 1024
 
 // Init placeholder when compiled without the 'secrets' build tag
-func Init(command string, arguments []string, timeout int, maxSize int) {}
+func Init(command string, arguments []string, timeout int, maxSize int, groupExecPrem bool) {}
```

Comment:
Group Exec **Permission** ?

```suggestion
func Init(command string, arguments []string, timeout int, maxSize int, groupExecPerm bool) {}
```